tests/post/steps/conftest.py

```diff
@@ -37,10 +54,11 @@ def _verify_kubeapi_service(host):
         assert retcode == 0
 
 
-def _run_bootstrap(host, version):
-    cmd = "/srv/scality/metalk8s-{0}/bootstrap.sh".format(
-        version['short_version']
-    )
+def _run_bootstrap(request, host):
+    # FIXME: this can only run on the bootstrap node, we'd need to skip such
+    #        test if the host fixture is not adapted
```

Comment:
or [limit the tests](https://testinfra.readthedocs.io/en/latest/invocation.html) on the bootstrap by adding 
`testinfra_hosts = ['bootstrap']`

Reply:
Good idea, thanks. I created #848 to track this.